indico/modules/events/tracks/models/tracks.py

```diff
@@ -103,6 +122,10 @@ class Track(DescriptionMixin, db.Model):
     # - contributions (Contribution.track)
     # - proposed_abstract_reviews (AbstractReview.proposed_tracks)
 
+    @property
+    def is_track_group(self):
```

Comment:
I don't know but I don't like this. Maybe you could just use `if item is instanceof Track` in your templates? You would still have to pass the `Track` class as a kwargs to the template